pyproject.toml

```diff
@@ -20,8 +20,11 @@ python-semantic-release = "^7.33.4"
 pre-commit = "^3.3.2"
 poethepoet = "^0.20.0"
 pytest-xdist = "^3.5.0"
+puyapy = { version = "^4.1.1", python = "^3.12" }
+algorand-python = { version = "^2.4.1", python = "^3.12" }
 setuptools = "^75.8.0"
 pytest-sugar = "^1.0.0"
+inflection = "^0.5.1"
```

Comment:
@CiottiGiorgio do you really need an whole dependency for a single to camel case use case? Also looks like the dependency is not actively maintained https://github.com/jpvanhal/inflection (no commits since 2020)

Reply:
@CiottiGiorgio see utils.py file in src, there are various formatters and sanitizers you can reuse, for instance you can make to_camel_case by reusing the to_pascal_case method and making first char lowercase?

Reply:
```
def to_camel_case(value: str) -> str:
    pascal_case = to_pascal_case(value)
    return pascal_case[0].lower() + pascal_case[1:]
```

Reply:
Good catch